### ISSUE TITLE
CSV import issues

### ISSUE THREAD

**Issue by locadani** [kuzudb/kuzu#2720]
Hello, I am trying to run the Python Colab examples (which use kuzu version 0.0.7) on my machine (which uses kuzu 0.1.0), and I have encountered problems when importing the CSV files, i.e. when running
`conn.execute('COPY tableName FROM "./fileName.csv" (HEADER=true)')`
Specifically, the problems arise in two examples: the node property prediction example ([link](https://colab.research.google.com/drive/1fzcwBwTY-M19p7OOTIaynfgHFcAQo9NK)), and the link prediction example ([link](https://colab.research.google.com/drive/1QdX7CDdajIAb04lqaO5PfJlpKG-ljG28)).

Both times the errors are due to some characters in the csv files: 
- for the Node prediction example, the error is in the paper.csv file; 
- for the Link prediction example, the error is in the movies.csv file.
I could fix the movies.csv file, by simply replacing the name of the movie with one that does not contain special characters, while I couldn't figure out the solution to the Node prediction example.

This is what I tried and learned:
- If I downgrade kuzu version to 0.0.7 or 0.0.8, the error is somehow solved and the code runs until the end; 
- If I downgrade kuzu to version 0.0.9, the CSV import works, but the code doesn't run until the end. This line fails `conn.execute("MATCH (a:paper) WHERE a.id = $pid SET a.category = $pred", [("pid", paper_id), ("pred", prediction)])`
- if i use version 0.1.0 I can load elements in the database as long as the rows of the csv file (including the header) are 22;
versions in between 

I hope this can be helpful, and we can find a solution.

**Comment by acquamarin** [comment]
Hi @locadani 
Thanks for letting us know the issue. 
1. For movies.csv:
We changed our default escape character from `"` to `\` recently, causing the movies.csv copy to fail. 
In movies.csv:
```
7789,2002,"11'09""01 - September 11 (2002)",Drama
```
This line uses the `"` as the default escape character.
To address the issue, you can manually specify the escape character in the copy statement:
```
COPY movie FROM "/Users/z473chen/Downloads/movies.csv" (HEADER=true, ESCAPE='"')
```
Our team is discussing whether we should change our default escape character back to `"`.

2. For paper.csv:
It is an known issue in the last release, and has already been fixed in the master. You can try it using our latest master, or wait for the python daily build package.

Hope it helps.
Thanks

**Comment by locadani** [comment]
Thank you for your answer, it was helpful.
Concerning the Node Prediction example, I updated my version to the latest available (0.1.1.dev36) and that problem was solved.
I just want to point out a few other modifications I had to make.

- The last rows of the file `paper.csv` do not have a category, so the code throws an error since there can't be NULL entries. I solved it adding some random numbers
- This split `train_count = int(0.6 * total_count)
test_count = int(0.4 * total_count)` gives `train_count` and `test_count` that don't add up to the `total_count`, so the program throws an error. I solved it by manually adding `+1`.

I hope this can help
